### ISSUE TITLE
Julia 1.0.0 - method struct error

### ISSUE THREAD

**Issue by niclaspopp** [fonsp/Pluto.jl#504]
Hey all,

I'm on Julia v1.0 and Windows 10 which should technically be supported by Pluto.jl. However, when trying to import Pluto I get the following Error message of which I can't really make sense of:

```
[ Info: Precompiling Pluto [c3e4b0f8-55cb-11ea-2926-15256bba5781]
ERROR: LoadError: LoadError: BoundsError: attempt to access 0-element Array{Symbol,1} at index [1]
Stacktrace:
 [1] getindex at .\array.jl:731 [inlined]
 [2] first at .\abstractarray.jl:270 [inlined]
 [3] |>(::Array{Symbol,1}, ::typeof(first)) at .\operators.jl:813
 [4] top-level scope at none:0
 [5] include at .\boot.jl:317 [inlined]
 [6] include_relative(::Module, ::String) at .\loading.jl:1038
 [7] include at .\sysimg.jl:29 [inlined]
 [8] include(::String) at C:\Users\Niclas Popp\.julia\packages\Pluto\upwxH\src\Pluto.jl:11
 [9] top-level scope at none:0
 [10] include at .\boot.jl:317 [inlined]
 [11] include_relative(::Module, ::String) at .\loading.jl:1038
 [12] include(::Module, ::String) at .\sysimg.jl:29
 [13] top-level scope at none:2
 [14] eval at .\boot.jl:319 [inlined]
 [15] eval(::Expr) at .\client.jl:389
 [16] top-level scope at .\none:3
in expression starting at C:\Users\Niclas Popp\.julia\packages\Pluto\upwxH\src\runner\PlutoRunner.jl:162
in expression starting at C:\Users\Niclas Popp\.julia\packages\Pluto\upwxH\src\Pluto.jl:37
ERROR: Failed to precompile Pluto [c3e4b0f8-55cb-11ea-2926-15256bba5781] to C:\Users\Niclas Popp\.julia\compiled\v1.0\Pluto\OJqMt.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] macro expansion at .\logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at .\loading.jl:1184
 [4] _require(::Base.PkgId) at .\logging.jl:311
 [5] require(::Base.PkgId) at .\loading.jl:852
 [6] macro expansion at .\logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at .\loading.jl:834
```

**Comment by fonsp** [comment]
Thanks!

Looks like this is `1.0.0`, and this line:
https://github.com/fonsp/Pluto.jl/blob/v0.11.14/src/runner/PlutoRunner.jl#L162
was only tested on `1.0.5` and up

**Comment by fonsp** [comment]
Basically we never supported 1.0.0 but according to our Project.toml we did. We can remove it from our Project.toml but then people with 1.0.0 will just install an old version of Pluto that is broken -- the only real solution is to actually fix support for 1.0.0

**Comment by fonsp** [comment]
On the Julia slack:

Fons van der Plas:pluto:  19 hours ago
Pluto never supported julia 1.0.0, only 1.0.4 and up (tests). I fixed this in my latest update to Project.toml, but can we also change this retroactively?

Fons van der Plas:pluto:  19 hours ago
Otherwise 1.0.0 users will get a broken, outdated version of Pluto, instead of an error that their julia is too old

Fons van der Plas:pluto:  19 hours ago
I did try to do this manually but I am afraid that I will make a mistake

Fons van der Plas:pluto:  19 hours ago
(and adding support for 1.0.0 is too difficult)

Mosè Giordano  19 hours ago
yes
:heartbeat:
1


Fredrik Ekre  19 hours ago
They also have a broken version of Julia so :shrugging:
:troll:
1


Fredrik Ekre  19 hours ago
I wouldn't approve such a change btw
